rdkit/Chem/UnitTestQED.py

```diff
@@ -0,0 +1,259 @@
+from __future__ import print_function
+
+from collections import namedtuple
+import os.path
+import doctest
+import unittest
+
+from rdkit import Chem
+from rdkit import RDConfig
+from rdkit.Chem import QED
+
+doLong = False
+TestData = namedtuple('TestData', 'lineNo,smiles,mol,expected')
+
+
+def load_tests(loader, tests, ignore):
+  """ Add the Doctests from the module """
+  tests.addTests(doctest.DocTestSuite(QED, optionflags=doctest.ELLIPSIS))
+  return tests
+
+
+class TestCase(unittest.TestCase):
+  pass
+  def test_properties(self):
+    m = Chem.MolFromSmiles('N=C(CCSCc1csc(N=C(N)N)n1)NS(N)(=O)=O')
+    p = QED.properties(m)
+    self.assertEqual([], p)
+    
+
+# 
```

Comment:
That is for sure. What worries me is that there is an obvious error in this test, `p != []`, but the travis-CI run didn't fail. (OMG - just found my mistake).

As I mentioned, the old doctests fail on Python 3.6. I've made several attempts to install 2.7 in an ubuntu machine and failed miserably. Well, eventually installed rdkit 2017.03 from conda and were able to test. The doctests will also fail on 2.7. Is Novartis still using a different definition of rotatable bonds @bp-kelley ? If yes, that's the reason. 

I'm not really concerned about the impact of having the number of rotatable bonds by a few. The differences in the qed values was 0.02. That's negligible for a qualitative descriptor like this. To make sure that we use a consistent approach, we better replace the `Lipinski.NumRotatableBonds(mol)` with a more suitable function.  Would directly using `rdMolDescriptors.CalcNumRotatableBonds(x)` be sufficient?